astroquery/mast/core.py

```diff
@@ -1985,7 +1984,7 @@ def query_region_async(self, coordinates, radius=0.2*u.deg, catalog="Hsc",
 
     @class_or_instance
     def query_object_async(self, objectname, radius=0.2*u.deg, catalog="Hsc",
-                           pagesize=None, page=None, **kwargs):
+                           version=None, pagesize=None, page=None, **kwargs):
```

Comment:
Order of kwargs might matter here as they can be used as positional args. New ones sadly needs to be added to the end until we finally switch to python3 only and keyword only arguments.

Reply:
That is sad, but also make sense. Fixed.